data/formats-data.js

```diff
@@ -3765,7 +3772,7 @@ exports.BattleFormatsData = {
 		tier: "PU",
 	},
 	kricketot: {
-		randomBattleMoves: ["endeavor", "mudslap", "bugbite", "strugglebug"],
+		randomBattleMoves: ["endeavor", "stringshot", "bugbite", "strugglebug"],
```

Comment:
i hope i never get this in random battles but uproar > string shot

Reply:
heh
sure